### ISSUE TITLE
Double-click on Mod to open files

### ISSUE THREAD

**Issue by erri120** [Nexus-Mods/NexusMods.App#1449]
Requires #1441, similar to #1439.

**Comment by BearWithMe18** [comment]
Is the intention here just to open the mod file in the in-app window like the current "View Mod Files" button? If so I can take this up.

**Comment by halgari** [comment]
Yep, for now it can do exactly the same as that button does, this is just
ease of use thing

**Comment by erri120** [comment]
> Is the intention here just to open the mod file in the in-app window like the current "View Mod Files" button? If so I can take this up.

@BearWithMe18 I've looked into this before and haven't resolved this issue yet, because of how we use the `DataGrid`. This issue sadly isn't as straightforward as it appears to be.
